moz_telemetry/io_modules/decoders/moz_telemetry/ping.lua

```diff
@@ -478,6 +478,7 @@ function transform_message(hsr)
         msg.Fields.geoCity         = hsr:read_message("Fields[geoCity]")
         msg.Fields.submissionDate  = os.date("%Y%m%d", hsr:read_message("Timestamp") / 1e9)
         msg.Fields.sourceName      = "telemetry"
+        msg.Fields.pingSenderVersion = hsr:read_message("Fields[X-PingSender-Version]")
```

Comment:
I am not big on changing the name.  I would prefer if the key stayed 'X-PingSender-Version'
@mreid-moz  may feel differently